bundles/org.openhab.binding.radiobrowser/src/main/java/org/openhab/binding/radiobrowser/internal/RadioBrowserBindingConstants.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.radiobrowser.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * The {@link RadioBrowserBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Matthew Skinner - Initial contribution
+ */
+@NonNullByDefault
+public class RadioBrowserBindingConstants {
+    private static final String BINDING_ID = "radiobrowser";
+    public static final int HTTP_TIMEOUT_SECONDS = 50;
```

Comment:
Is there a specific reason to keep the timeout that long? I would say a timeout for 15 seconds is allready very long.

Reply:
When I first started using the API I had it set to 10 seconds and got a timeout. It may have been a server update in progress. I have put it back to 10 and will do some smoke tests after having made all your suggestions.